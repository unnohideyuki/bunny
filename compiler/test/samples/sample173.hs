main = do print $ take 10 (enumFrom 2 :: [Int])
          print $ (enumFromTo 3 9 :: [Int])
          print $ take 10 (enumFromThen (-1) 3 :: [Int])
          print $ (enumFromThenTo 1 3 8 :: [Int])

          print $ (enumFromTo 1 0 :: [Int])
          print $ (enumFromThenTo 1 2 0 :: [Int])
          print $ take 10 $ (enumFromThenTo 1 1 2 :: [Int])
          print $ take 10 $ (enumFromThenTo 1 1 1 :: [Int])
          print $ take 10 $ (enumFromThenTo 1 1 0 :: [Int])
