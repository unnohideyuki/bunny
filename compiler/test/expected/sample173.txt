[2,3,4,5,6,7,8,9,10,11]
[3,4,5,6,7,8,9]
[-1,3,7,11,15,19,23,27,31,35]
[1,3,5,7]
[]
[]
[1,1,1,1,1,1,1,1,1,1]
[1,1,1,1,1,1,1,1,1,1]
[]
